cloud/jenkins/psmdb_operator_gke_version.groovy

```diff
@@ -290,7 +290,9 @@ pipeline {
                         CreateCluster('selfhealing')
                         runTest('storage', 'selfhealing')
                         runTest('self-healing', 'selfhealing')
+                        runTest('self-healing-chaos', 'selfhealing')
                         runTest('operator-self-healing', 'selfhealing')
+                        runTest('operator-self-healing-chaos', 'selfhealing')
                         ShutdownCluster('selfhealing')
```

Comment:
no self-healing-advanced-chaos ?

Reply:
it seems that for PSMDB that test doesn't exist